scripts/jenkins/init-node.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Install nvm for this shell
+source ~/.nvm/nvm.sh
+
+# Use version of node.js found in .nvmrc
+nvm install
+
+npm install -g yarn
```

Comment:
Please avoid installing Yarn via npm if possible. This is significantly slower than all other methods of installing Yarn, and often results in weird issues around Yarn's dependencies. We (Yarn team) do not recommend it

If running on Debian or Ubuntu, you should use Yarn's Debian repository. Otherwise, use the installation script (`curl -o- -L https://yarnpkg.com/install.sh | bash`) or just extract the tarball manually. Both methods (Debian package repository and installation script) allow you to specify the exact version of Yarn to install.